### DISCUSSION TITLE
API to control and add music

### DISCUSSION THREAD

**Discussion by nglmercer** [KRTirtho/spotube#1489] [Q&A] [answered]
Is there a way to control the music with some API. I would like to use it to control it from another device and add music with voice commands if this is possible.
### Developer features or remote access implementation

**Comment by KRTirtho** [comment, marked as answer]
Yup, you can do that using, the new Connect Feature.

Basically, enable "Settings" > "Enable Connect" on host devices and from the client/control device go to "Home" > "Devices" and you will see it in the List as long as both are in same WIFI/network

**Reply by KRTirtho** [reply]
We, use WebSocket under the hood and zeroconf for device discovery. You can use this APIs to control the playback programatically

**Comment by nglmercer** [comment]
Thanks, but it would still be good to include some information about what parameters it receives, the routes and all that

**Reply by KRTirtho** [reply]
This is not quite stable yet and I'm still figuring out what will be the best approache. So, I didn't document them yet
